templates/participants.html

```diff
@@ -0,0 +1,13 @@
+{% extends "right-aside.html" %}
+{% block subhead_content%}
+{% endblock subhead_content%}
+
+{% block main_content %}
+{{ page.content | safe }}
+
+{% include "includes/participants-buttons.html" %}
+<a href="https://learn.percona.com/percona-support-for-valkey" class="logo">
```

Comment:
Maybe make it data driven?

We could have a `participants.yml` (or toml, whatever) file and loop over the items freeing anyone from having to do HTML (and keeping it consistent).

For an example of something similar look at how the repos on the home page are done (https://github.com/valkey-io/valkey-io.github.io/blob/main/templates/index.html#L38) which is driven from the [`repos.yml`](https://github.com/valkey-io/valkey-io.github.io/blob/main/_data/repos.yml) file. 

Glad to walk through it more if needed.